### PR TITLE
auto OCP-25931 

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  could you help to review the cases? Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/operator_hub.xyaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+create_catalog_source:
+  action: goto_catalog_source_page
+  action: click_button
+  action: input_catalog_source_name
+  action: input_display_name
+  action: input_publisher_name
+  action: input_image
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/operator_hub.xyaml]: It looks like click create button is missing, the last step

File: lib/rules/web/admin_console/4.3/operator_hub.xyaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  params:
+    input_field_id: catalog-source-publisher
+    input_value: <publisher-name>
+  action: set_input_value
+input_image:
+  params:
+    input_field_id: catalog-source-image
+    input_value: <image>
+  action: set_input_value
+delete_one_catalog_source:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/operator_hub.xyaml]: Can we reuse something from `delete_project` ?

**Comment by SSshahan** [comment]
@yapei  could you help to review again?

**Comment by SSshahan** [comment]
@pruan-rht  could you help to merge it?

**Pull request merged**